src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/PrivateDataRetrievalService.java

```diff
@@ -53,7 +56,13 @@ private ManagingOfficerListDataApi getManagingOfficerData(String companyNumber)
 
       ApiLogger.debug("Retrieving Managing Officer data for Company Number ", logMap);
       return managingOfficerDataList;
-
+    } catch (ApiErrorResponseException e) {
+      if (e.getStatusCode() == HttpServletResponse.SC_NOT_FOUND) {
+        ApiLogger.info(String.format("No Managing Officers data found for Company Number %s", companyNumber), logMap);
+        var managingOfficerDataList = new ManagingOfficerListDataApi(Collections.emptyList());
```

Comment:
Need to log error before re-throwing. That's what we do for general errors on line 67 to 69.